src/pages-chibi/implementations/Comikey/main.ts

```diff
@@ -0,0 +1,124 @@
+          $c.url().urlPart(6).boolean().run(),
+          $c.url().urlPart(6).matches('(?:chapter|episode|espanol|bahasa|portugues)[_-]?').run(),
+        )
+        .run();
+    },
+    getTitle($c) {
+      return $c.querySelector('.br-toolbar__ellipsis').text().split(' (').at(0).trim().run();
+    },
+    getIdentifier($c) {
+      return $c.url().urlPart(4).replaceRegex('-[^-]+$', '').split('-pt').at(0).run();
+    },
+    getImage($c) {
+      return $c.querySelector('.logo img').getAttribute('src').urlAbsolute().ifNotReturn().run();
+    },
+    getOverviewUrl($c) {
+      return $c
+        .querySelector('.br-toolbar__ellipsis')
+        .getAttribute('href')
+        .urlAbsolute()
+        .ifNotReturn()
+        .run();
+    },
+    getEpisode($c) {
+      return $c
+        .url()
+        .urlPart(6)
+        .regex('(?:chapter|episode|espanol|bahasa|portugues)[_-]?(\\d+)', 1)
+        .number()
+        .run();
+    },
+    nextEpUrl($c) {
+      return $c.querySelector('.next a').getAttribute('href').ifNotReturn().urlAbsolute().run();
+    },
+    readerConfig: [
+      {
+        condition: '#br-spine canvas',
+        current: {
+          selector: '#br-spine canvas',
+          mode: 'countAbove',
+        },
+        total: {
+          selector: '#br-spine canvas',
+          mode: 'count',
+        },
+      },
+      {
+        condition: '#br-spine img',
+        current: {
+          selector: '#br-spine img',
+          mode: 'countAbove',
+        },
+        total: {
+          selector: '#br-spine img',
+          mode: 'count',
+        },
+      },
+    ],
+  },
+  overview: {
+    isOverviewPage($c) {
+      return $c
+        .and(
+          $c.url().urlPart(4).boolean().run(),
+          $c.url().urlPart(3).equals('comics').run(),
+          $c.url().urlPart(6).boolean().not().run(),
+        )
+        .run();
+    },
+    getTitle($c) {
+      return $c.querySelector('.sub-data .title').text().split(' (').at(0).trim().run();
+    },
+    getIdentifier($c) {
+      return $c.this('sync.getIdentifier').run();
+    },
+    getImage($c) {
+      return $c.querySelector('[property="og:image"]').getAttribute('content').ifNotReturn().run();
+    },
+    uiInjection($c) {
+      return $c.querySelector('.description').uiAppend().run();
+    },
+  },
+  list: {
+    elementsSelector($c) {
+      return $c.querySelectorAll('#chapters .chrow').run();
+    },
+    elementUrl($c) {
+      return $c.getAttribute('href').urlAbsolute().run();
+    },
+    elementEp($c) {
+      return $c.this('list.elementUrl').this('sync.getEpisode').run();
+    },
+  },
+  lifecycle: {
+    setup($c) {
+      return $c.addStyle(require('./style.less?raw').toString()).run();
+    },
+    ready($c) {
+      return $c.detectURLChanges($c.trigger().run()).domReady().trigger().run();
+    },
+    listChange($c) {
+      return $c.detectChanges($c.querySelector('#chapters').text().run(), $c.trigger().run()).run();
```

Comment:
Add a `.ifNotReturn()` after the querySelector to remove the error at least. There is somekind of bug that it still triggers, but that is not your Problem